<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Таблицы для Статики</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonLoad">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Загрузить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonUnload">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Выгрузить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelCode_2">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Количество строк</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="boxCountTable">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>990</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="label1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTableWidget" name="table">
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Буфер</string>
    </property>
    <addaction name="menuLoad"/>
    <addaction name="menuUnload"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="menuAbout"/>
    <addaction name="menuQt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="menuUnload">
   <property name="text">
    <string>Выгрузить</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="menuAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="menuQt">
   <property name="text">
    <string>O Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
